modules/dynatrace-activegate/vm.tf

```diff
@@ -1,5 +1,5 @@
 locals {
-  prefix    = "activegate-${var.env}"
+  prefix    = "activegate"
```

Comment:
this will cause a rebuild and downtime of dynatrace, is that ok / expected?

Reply:
I have reverted, need to make some more changes and closed the PR @timja